.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: elixir
+elixir:
+  - 1.4.1
+addons:
+  postgresql: '9.4'
```

Comment:
I still this travis.yml from @shouston3 & @jackcarlisle so you need to blame them :wink: 
There is Postress 9.6: 
![postgres-version](https://cloud.githubusercontent.com/assets/6057298/23710940/4d1cfe1c-0416-11e7-847f-d42f024f7270.png)

I'm not sure if we need "sudo enable" so that's might be a reason that 9.4 is used to be sure that everything is working.

also ```The Travis CI build environments use version 9.1```